src/TestExplorer/TestRunner.ts

```diff
@@ -40,6 +40,8 @@ import { BuildConfigurationFactory, TestingConfigurationFactory } from "../debug
 import { TestKind, isDebugging, isRelease } from "./TestKind";
 import { reduceTestItemChildren } from "./TestUtils";
 import { CompositeCancellationToken } from "../utilities/cancellation";
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+import stripAnsi = require("strip-ansi");
```

Comment:
nit: I think we have ES module support enabled in our TS config which should allow you to do something like:
```typescript
import * as stripAnsi from "strip-ansi";
```
Either that or disable the no-require-imports rule in our eslint config. I'm not a fan of adding es lint ignores everywhere.

Reply:
We don't have esModuleInterop set in our tsconfig.json. I think we could try turning that on, but I'd like to do it in a follow on patch.

Reply:
Whoops, thought we already had that enabled. Yeah I think that's best turned on in another patch.